gulpfile.js

```diff
@@ -57,7 +57,7 @@ gulp.task('dev', (done) => {
 gulp.task('watch', function() {
     browserSync.init({
         server: {
-            baseDir: "./www",
+            baseDir: "./src",
```

Comment:
Since a recent merge, the watch task need to serve from the `www` folder. Is this a PR conflict?